src/integration/pages/storyPage/canonicalTests.js

```diff
@@ -0,0 +1,25 @@
+import {
+  runCoreCanonicalTests,
+  runCanonicalAnalyticsTests,
+} from '../../common';
+
+export default () => {
+  runCoreCanonicalTests();
+  runCanonicalAnalyticsTests();
+
+  describe('Social Embeds', () => {
+    const hasRichInstagramEmbed = !!document.querySelector(
+      'iframe.instagram-media',
+    );
+
+    if (hasRichInstagramEmbed) {
+      it("I can see the social media provider's JavaScript", () => {
```

Comment:
Is it worth specifying Instagram in the test description?

Reply:
Do you think it's necessary to replicate this test for all supported providers? https://github.com/bbc/simorgh/pull/6302#discussion_r419550220 is somewhat related.

Reply:
Mmmm not sure? This is fine for now, but maybe if there are other examples, it could be good to include them at a later date? (Let's maybe wait until we have integration tests for all services, and see what's missing then)

Reply:
Re the discussion above, at integration test level, I'm not sure we want comprehensive coverage but lean testing of most functionality, unit testing should be comprehensive I think. This approach of only testing instagram looks fine to me.